app/services/permissions_service.rb

```diff
@@ -43,6 +46,7 @@ def guest_permissions
       return true if controller == "skills" && action == "show"
       return true if controller == "professionals" && action == "index"
       return true if controller == "sessions" && action.in?(%w(new create))
+      # a registered user should not be authorized for line 50
       return true if controller == "users" && action.in?(%w(new create))
```

Comment:
A group discussion about how we're handling permissions might be good at some point! They affect everything else on the project. Would that be okay? @alishersadikov @AliSchlereth @meganft